src/reducers/inputs.ts

```diff
@@ -0,0 +1,28 @@
+import { Action } from 'redux';
+
+interface ActionWithPayload extends Action {
+  payload: any
+}
+
+const inputs = (state = [], {type, payload}: ActionWithPayload) => {
```

Comment:
you don't need this reducer, cause you would only consume inputs not modify them